web-exercise/ex-5/script.js

```diff
@@ -0,0 +1,179 @@
+async function fetch_pokemons(url){
+        const response=await fetch(url);
+        const data= await response.json();
+        console.log(data);
+        show_on_screen(data);
+}
+async function fetch_pokemon_details(pok){
+    const response=await fetch(pok.url);
+    const data= await response.json();
+    console.log(data);
+    store_data.push(data);
+
+    render_card(data);
+            
+}
+function render_card(data){
+    const el = document.createElement('div');
+    const image = document.createElement('img');
+    const btn = document.createElement('button');
+    btn.textContent="Read More";
+    // btn.onclick=show_details(data,pok);
+
+    btn.addEventListener("click", ()=>{
+        // let frameUrl="frame.html";
+        // let page=window.open(frameUrl, "description");
+        // let details= page.document.getElementById("details");
+        // details.textContent="abcd";
+        
+        const iframeDocument = document.querySelector("iframe").contentDocument;
+        iframeDocument.body.style="font-size:20px;padding:35px;background-color:black;color:white;";
+       let h;
+       h="180px";
+        iframeDocument.body.innerHTML = "<div><h2>Pokemon Details</h2><img height="+h+" src='"+data.sprites.front_shiny+"'><img src='"+data.sprites.back_shiny+"'><div>id: "+data.id+"</div><div>Name:"+data.name+"</div><div>base_experience: "+data.base_experience+"</div><div>height: "+data.height+"</div><div>Stats: "+data.stats[0].base_stat+"</div><div>Type: "+data.types[0].type.name+"</div></div>";
+    });
+    
+    image.src=data.sprites.front_default;
+    image.addEventListener("mouseout", ()=>{
```

Comment:
Use the `alt` attribute to describe the image. Read more [Write good Alt Text to describe images](https://accessibility.huit.harvard.edu/describe-content-images#:~:text=Alternative%20(Alt)%20Text%20is%20meant,example%20of%20a%20missing%20image.)